### DISCUSSION TITLE
Sample Tumor Status - CDE 5432687

### DISCUSSION THREAD

**Discussion by GeoffLyle** [CBIIT/ccdi-federation-api#39] [API Design Discussion]
caDSR Link: [5432687](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=5432687%20and%20ver_nr=2)

Definition: "Text term that represents a description of the kind of tissue collected with respect to disease status or proximity to tumor tissue."

Approved CDE as of December 13th, 2023.

**Comment by GeoffLyle** [comment]
Per March 6, 2025 meeting:

CDE [5432687](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=5432687%20and%20ver_nr=2) has been retired.

Suggested use CDE [14688604](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=14688604%20and%20ver_nr=1).

The new CDE contains fewer permissible values than the one currently implemented.  Missing values **Abnormal**, **Non-neoplastic**, **Unavailable**, and **Unspecified**.  Looking at the current use for this CDE, implemented in field `tissue_type` only the values **Tumor** and **Normal** are being used, which is common to both the old and new CDE.

Will review whether Federation members have a use for the missing values and discuss implementing the CDE in the v2 release.

**Reply by mcrusch** [reply]
The suggested CDE works for us.  Note FYI that this was also one of the three recommended for biospecimens--since biospecimens will just be samples with additional annotations, when we update for samples broadly we'll get it for biospecimens for free.

**Comment by GeoffLyle** [comment]
Per March 20, 2025 meeting:

Federation members agreed the new CDE ([14688604](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=14688604%20and%20ver_nr=1)) is functionally similar to the current CDE.  The missing Permissible Values in the new CDE will not be a problem as values **Not Reported** and **Unknown** remain.  Currently, adopting the new CDE will not requires any harmonizing changes to the data shared.  The new CDE will be referenced in the implementation code as well as the GitHub documentation.  Will discuss how these changes will be made in the API Workgroup.
